### PR TITLE
Jobname support

### PR DESCRIPTION
**Pull request by yitzchak**
Add support for jobnames TeX Magic.  ``# !TEX jobnames = foo bar`` causes two builds. First with "foo" and second with "bar."

### CODE REVIEW

File: lib/composer.js [hunk 1]
```diff
@@ -42,26 +42,29 @@ export default class Composer {
...
         reject(statusCode)
       }
 
       try {
-        statusCode = await builder.run(rootFilePath)
-        result = builder.parseLogFile(rootFilePath)
-        if (statusCode > 0 || !result || !result.outputFilePath) {
-          showBuildError(statusCode, result, builder)
-          return
+        for (let jobname of jobnames) {
```
Comment (thomasjo) [inline lib/composer.js]: This `let` can be made `const` since it is not mutated.

**Comment by thomasjo** [comment]
This is looking really good!

Other than the little `const` comment I left, it would be awesome with a test that ensures the jobname loop works as expected. Essentially something that verifies that the builder is invoked the expected number of times. 

Having the aforementioned loop be an async loop would be a dream scenario, but I can sort that out at a later stage if you don't want to deal with that right now.

Thanks for helping out! 🙇

**Comment by yitzchak** [comment]
:cool: Thanks for looking at it!

I fixed the let/const issue. I'll work on a test for the jobname loop. It might take me a bit longer to turn it into an async loop. I am not really a javascript expert.

**Comment by thomasjo** [comment]
> It might take me a bit longer to turn it into an async loop. I am not really a javascript expert.

Don't worry about the async stuff. I can implement that at a later stage ⏩

**Comment by thomasjo** [comment]
Sorry about the terribly long delay on merging this! I resolved the merge conflicts since it was my fault for letting this linger for such a long time.

Thanks again for contributing! I'll try to deal with your [other PR](https://github.com/thomasjo/atom-latex/pull/186) as soon as possible 🙇

**Comment by yitzchak** [comment]
Awesome! Thanks for taking care of the conflicts and taking the PR.

**Pull request merged**